### ISSUE TITLE
Package names ignored

### ISSUE THREAD

**Issue by lanttu** [malijs/mali#9]
Mali ignores package names when defining services, e.g. `mali.use('TestService', ...)` instead of `mali.use('test.TestService' ...)`. I've not tested what happens if services in different packages have the same name, but nevertheless I think package names should be required in service definition process.

**Comment by bojand** [comment]
Hello, it took a bit long to look into this... In [0.2.0](https://github.com/malijs/mali/releases/tag/v0.2.0) release I added more params to `use` so we can specify package name and service name. So you can use

```
app.user('test', 'TestService', ...)
```

to specify the package `test`. It's fully documented [here](https://malijs.github.io/#/api?id=maliuse). Does this not satisfy the requirement? Maybe I am not completely understanding the issue? Feedback always welcome. Thanks.

**Comment by lanttu** [comment]
By packages I meant protobuf packages, ie. the package row in the beginning of the proto file something like `package test.sub1`, right after syntax row.

Maybe I need more coffee, but I failed to find any mention about packages in the docs. :) When I add a service implementation I have to provide service name (and method), but not the package name. If the protobuf definition I provided to Mali constructor contains multiple services with the same name the service names will conflict. Like in the scenario below.

```
| - package service1
|   | - EndpointService
| - package service2
|   | - EndpointService
```

**Comment by bojand** [comment]
Sorry maybe I needed more coffee hah. Don't know what I was saying in my comment lol... We can specify service name, _not_ package name... I'll have to investigate this. I am not sure if you can have more than 1 package that defines services in grpc, but if you can this could be an issue. Thanks.

**Comment by bojand** [comment]
Hello, this should be fixed in 0.14.0. You can use full package definition to reference a service:

```js
app.use('myorg.myapi.v1.MyService', 'getUser', mw1, mw2, getUser)
```
